core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2725,9 +2725,11 @@ struct ViewDataHandle<
                                    Kokkos::CudaUVMSpace>::value))
 #endif
                 && (!Traits::memory_traits::is_atomic))>::type> {
-  using value_type  = typename Traits::value_type;
-  using handle_type = typename Traits::value_type* KOKKOS_IMPL_ALIGN_PTR(
-      KOKKOS_MEMORY_ALIGNMENT);
+  using value_type = typename Traits::value_type;
+  // typedef work-around for intel compilers error #3186: expected typedef
+  // declaration
+  typedef typename Traits::value_type* KOKKOS_IMPL_ALIGN_PTR(
```

Comment:
Why `Traits::value_type*` when you alias value_type to Traits::value_type above? Could just be `value_type*`, no?

Reply:
Sure

Reply:
Yeah, I didn't look carefully when I replaced the using with typedef, clang-format pulled in the `value_type` line 2728 in (no apparent change). I'll rebase the change in, thanks

Reply:
This does not need to be addressed at this time.  My preference goes to merging your PR as you opened it.

Reply:
Just rebased the cleanup in